### PR TITLE
recipe for output_viewer

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: recipes/output_viewer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+about:
+  home: https://github.com/esgf/output_viewer
+  summary: The Output Viewer is designed to provide a framework for viewing arbitrary output from diagnostics scripts, metrics, or any program that produces a ton of different files that you want to look at it in an easy-to-use fashion.
+  license: ESGF
+  license_file: LICENSE
+
+extra:
+    recipe-maintainers:
+        - doutriaux1
+        - zshaheen
```
Comment (mariusvniekerk) [inline recipes/output_viewer/meta.yaml]: @zshaheen are you okay with being maintainer on this feedstock?

Comment (doutriaux1) [reply recipes/output_viewer/meta.yaml]: @mariusvniekerk thanks for asking. @zshaheen and I work together he is actually the one that uses this the most as part of his cdp package, which depnds on this and is coming down as a staged recipe as soon as this is a feedstock.

Comment (doutriaux1) [reply recipes/output_viewer/meta.yaml]: @mariusvniekerk by the way when you try to add a few packages with chained dependencies, is there a way to put them all in one staged-recipe? Or do we have to go through the process 3 times?

Comment (zshaheen) [reply recipes/output_viewer/meta.yaml]: @mariusvniekerk Yes, I'm fine with this!

**Pull request merged**